src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -164,6 +163,14 @@ static ItemStack of(ItemType itemType, int quantity) {
         Builder quantity(int quantity) throws IllegalArgumentException;
 
         /**
+         * Sets the ItemStack's quantity to represent any size to be used for
+         * querying inventories.
+         *
+         * @return This builder, for chaining
+         */
+        Builder anySize();
```

Comment:
Should there be a method called `empty` or `invalid` since Minecraft introduced invalid item stack concepts in 1.11?